llvm/lib/Target/X86/X86.td

```diff
@@ -1237,8 +1237,6 @@ def ProcessorFeatures {
   // Gracemont
   list<SubtargetFeature> GRTTuning = [TuningMacroFusion,
                                       TuningSlow3OpsLEA,
```

Comment:
LEA also seems to be reasonably fast on GRT.

Reply:
No sure. We have `TuningSlow3OpsLEA` for AlderLake and uops.info shows it has longer latency on it.

I don't have any information about newer targets.

Reply:
2c latency, but other slow lea is 3c. Bit of a middle ground.

Reply:
Interestingly, the latency was reduced to 1 since IceLake but was grown back to 2 on AlderLake.

I think we can change it for IceLake, but AlderLake etc. need more date provements.